src/groups/mqb/mqbblp/mqbblp_routers.t.cpp

```diff
@@ -72,18 +72,16 @@ struct TestStorage {
                 1,
                 d_domainCfg,
                 &d_capacityMeter,
-                bmqp::RdaInfo(),
                 allocator)
     , d_iterator(d_storage.getIterator(mqbu::StorageKey()))
     , d_bufferFactory(32, allocator)
-    , d_queue_sp(new (*allocator) mqbmock::Queue(0, allocator), allocator)
+    , d_queue_sp(new(*allocator) mqbmock::Queue(0, allocator), allocator)
```

Comment:
This might be a bug in the local clang-format
```suggestion
    , d_queue_sp(new (*allocator) mqbmock::Queue(0, allocator), allocator)
```

Reply:
The clang-format does without the space but the C++ Formatting Check does not pass so I had to manually insert the space.